glpi-official/services/web/files/opt/download_glpi.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
These scripts should exit as long as an error occurred.

```suggestion
#!/bin/bash -e
```